src/components/VideoPlayer/VideoPlayer.stories.mdx

```diff
@@ -49,25 +59,39 @@ The `aspectRatio` prop is used to control the sizing of the space allotted for t
   <Story
     name="VideoPlayer"
     args={{
-      videoType: VideoPlayerTypes.Vimeo,
-      videoId: "",
-      headingText: "NYPL Video",
+      aspectRatio: "VideoPlayerAspectRatios.FourByThree",
```

Comment:
The video that is first loaded has a 16:9 aspect ratio, so the `apectRatio` prop should match that. Change this to `SixteenByNine`.